src/dasi/tools/dasi_get.cc

```diff
@@ -0,0 +1,126 @@
+
+#include <algorithm>
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+#include <fstream>
+#include <memory>
+#include <sstream>
+#include <utility>
+#include <vector>
+
+#include "dasi/api/Dasi.h"
+#include "dasi/api/ReadHandle.h"
+
+#include "dasi/util/AutoCloser.h"
+#include "dasi/util/Exceptions.h"
+
+using dasi::util::UserError;
+
+
+dasi::api::Query parseQuery(const std::string& s) {
```

Comment:
We are going to need to parse queries elsewhere. This is in the wrong place.

Reply:
I added it here for now since it is the only place it is needed, but I move it to a separate file in [feature/list](https://github.com/ecmwf-projects/dasi/tree/feature/list)